src/styles/_figure.scss

```diff
@@ -42,6 +42,12 @@
             transform: rotate(1deg) scale(1.2);
         }
 
+        &.zoomable:active .image,
+        .image.zoomable:active {
+            transform:scale(1.0);
+            transition: 1s ease-out opacity, 0.1s ease-out transform, .2s ease-out margin;
+        }
```

Comment:
makes zoom transition (from previous comment) look more natural and gives some feedback to clicking the image  
without it, zoom transition seems kinda weird, it changes instantly to initial size, and then zooms in - and, once thumbnail loses focus it slowly shrinks back to normal size again, while full screen image is already open